docs/snippets/README.md

```diff
@@ -1,3 +1,5 @@
 # WooCommerce code snippets
 
 Various code snippets you can add to your site to enable various custom functionality.
```

Comment:
Suggest removing the superfluous `various` here:
```suggestion
Various code snippets you can add to your site to enable custom functionality:
```

Reply:
I pre-approved so when you make the suggested changes you can just go ahead and merge.